docs/designs/index.md

```diff
@@ -2,17 +2,17 @@
 
 ## Upcoming
 
-- [2022_02_13_comment_chip](2022_02_13_comment_chip.pdf)
-- [2022-01-25 Wire Flash Debugging](2022_01_25_wire_flash_debugging.pdf)
+- [2022-02-13 Comment Chip](2022_02_13_comment_chip.pdf)
 - [2022-01-25 Combat V2](2022_01_25_combat_v2.pdf)
 - [2022-01-25 Doors V2](2022_01_25_doors_v2.pdf)
-- [2021-11-15 Interaction Roles](2021_11_15_interaction_roles.pdf)
 - [2021-11-15 CV2 Color Type and Chips](2021_11_15_color_type_and_chips.pdf)
 - [2021-11-01 CV2 Environments](2021_11_01_cv2_environments.pdf)
 - [2021-11-01 Sampler](2021_11_01_sampler.pdf)
 - [2021-11-01 Set Position and Respawn](2021_11_01_set_position_and_respawn_support.pdf)
 
 ## Released
 
+- [2022-01-25 Wire Flash Debugging](2022_01_25_wire_flash_debugging.pdf)
```

Comment:
Treating this as released, but there may be parts of this that are not released. In that case, a commit may be added to move it back to upcoming.